db/seeds.rb

```diff
@@ -0,0 +1,9 @@
+# This file should ensure the existence of records required to run the application in every environment (production,
+# development, test). The code here should be idempotent so that it can be executed at any point in every environment.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Example:
+#
+#   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
+#     MovieGenre.find_or_create_by!(name: genre_name)
+#   end
```

Comment:
it's good to add seeds to test your project rather than adding from the console 
`messages = Message.create([ { greeting: "Hello World! 1" }, { greeting: "Goodbye World!1" }, { greeting: "Hello Goodbye World!2" }, { greeting: "Goodbye Hello World!2" } ])`